tests/molecules/LiH_dimer_pp/det_vmc_hf_noj.in.xml

```diff
@@ -2441,11 +2441,10 @@ GAMESS numerical results:
    </qmcsystem>
    <qmc method="vmc" move="pbyp">
       <parameter name="walkers"             >    1               </parameter>
-      <parameter name="blocks"              >    4             </parameter>
-      <parameter name="steps"               >    5            </parameter>
-      <parameter name="subSteps"            >    3               </parameter>
+      <parameter name="blocks"              >    3             </parameter>
+      <parameter name="steps"               >    3            </parameter>
+      <parameter name="subSteps"            >    1               </parameter>
```

Comment:
1 "actual substep" means substeps=2 here.  Please update.

Reply:
Current references are using AoS/real/full precision/cpu build. It is same with the build you suggested. I'll update the substep.